src/api/custom-schema/type-defs.ts

```diff
@@ -229,6 +234,40 @@ const typeDefs = `
     allCurrencies: [MappedCurrency]
   }
 
+  type Owner {
+    firstName: String
+    lastName: String
+    email: String
+  }
+
+  type FullApplication {
```

Comment:
Any reason for `FullApplication`, maybe just `Application`? If not I think `PopulatedApplication` would be more suitable, what do you think?

Reply:
Can do PopulatedApplication.  Cannot do Application as it messes with the autogenerated graphql Application